style.css

```diff
@@ -0,0 +1,123 @@
+/* @import url('https://fonts.cdnfonts.com/css/poppins'); */
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  margin: unset;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+#logo {
+  /* width: 88%; */
+
+  /* width: 76px; */
+
+  /* height: 20px; */
+
+  /* height: 28%; */
+  margin-top: 77.5%;
+
+  /* margin-top: 58px; */
+  margin-left: 50%;
+
+  /* margin-left: 35px; */
+}
+
+.fa-solid {
+  /* width: 18px; */
+
+  /* height: 12px; */
+  color: #6070ff;
+  margin-right: 1.95rem;
+  margin-top: 62px;
+  margin-top: 3.55rem;
+  font-size: 18px;
+}
+
+.back-image {
+  background-image: url("images/header.jpg");
```

Comment:
- [Optional] Kindly edit url, change `images` to `Images` so the background can display properly on all ends.

Reply:
Done.